x-pack/test/functional/apps/dashboard/sync_colors.ts

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { DebugState } from '@elastic/charts';
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default function ({ getService, getPageObjects }: FtrProviderContext) {
+  const esArchiver = getService('esArchiver');
+  const PageObjects = getPageObjects([
+    'common',
+    'dashboard',
+    'spaceSelector',
+    'header',
+    'lens',
+    'timePicker',
+  ]);
+  const dashboardAddPanel = getService('dashboardAddPanel');
+  const filterBar = getService('filterBar');
+  const elasticChart = getService('elasticChart');
+
+  function getColorMapping(debugState: DebugState | null) {
+    if (!debugState) return {};
+    const colorMapping: Record<string, string> = {};
+    debugState.bars?.forEach(({ name, color }) => {
+      colorMapping[name] = color;
+    });
+
+    return colorMapping;
+  }
+
+  describe('sync colors', function () {
+    before(async function () {
+      await esArchiver.loadIfNeeded('logstash_functional');
+      await esArchiver.loadIfNeeded('lens/basic');
+    });
+
+    after(async function () {
+      await esArchiver.unload('logstash_functional');
+      await esArchiver.unload('lens/basic');
+    });
+
+    it('should sync colors on dashboard by default', async function () {
+      await PageObjects.common.navigateToApp('dashboard');
+      await PageObjects.dashboard.clickCreateDashboardPrompt();
+      await dashboardAddPanel.clickCreateNewLink();
+      await dashboardAddPanel.clickVisType('lens');
+      await PageObjects.header.waitUntilLoadingHasFinished();
+      await PageObjects.lens.goToTimeRange();
+
+      await PageObjects.lens.configureDimension({
+        dimension: 'lnsXY_yDimensionPanel > lns-empty-dimension',
+        operation: 'count',
+        field: 'Records',
+      });
+
+      await PageObjects.lens.configureDimension({
+        dimension: 'lnsXY_splitDimensionPanel > lns-empty-dimension',
+        operation: 'terms',
+        field: 'geo.src',
+      });
+
+      await PageObjects.lens.save('vis1', true, true);
+      await PageObjects.header.waitUntilLoadingHasFinished();
+      await dashboardAddPanel.clickCreateNewLink();
+      await dashboardAddPanel.clickVisType('lens');
+      await PageObjects.header.waitUntilLoadingHasFinished();
+      await PageObjects.lens.goToTimeRange();
```

Comment:
Is the time range cleared in between the previous page load and this one? I would have expected it to persist.